### PR TITLE
Aqueduct remote runways

### PR DESCRIPTION
**Pull request by blkerby**

### CODE REVIEW

File: region/maridia/inner-pink/Aqueduct.json [hunk 1]
```diff
@@ -887,6 +988,79 @@
         {"shinespark": {"frames": 40, "excessFrames": 7}}
       ]
     },
+    {
+      "link": [2, 7],
+      "name": "Temporary Blue Gravity Jump",
+      "requires": [
+        "h_canShineChargeMaxRunway",
+        "canGravityJump",
+        "canChainTemporaryBlue"
```
Comment (osse101) [inline region/maridia/inner-pink/Aqueduct.json:997]: HiJump should be enough to only use a regular tempblue grav jump

File: region/maridia/inner-pink/Aqueduct.json [hunk 2]
```diff
@@ -1529,6 +2032,35 @@
...
+      "name": "Temporary Blue Chain",
+      "entranceCondition": {
+        "comeInWithTemporaryBlue": {
+          "direction": "left"
+        },
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        "canChainTemporaryBlue",
+        "can4HighMidAirMorph"
```
Comment (osse101) [inline region/maridia/inner-pink/Aqueduct.json:2046]: With gravity off, the jump morphs aren't tight

File: region/maridia/inner-pink/Aqueduct.json (hunk 1 above)
Comment (blkerby) [reply region/maridia/inner-pink/Aqueduct.json]: Yeah, good find, it does work.

**Review by osse101 — COMMENTED** [review]
I tried out the [2,1] strat.  The [2,5] double springball jump is on a bit of the harder side for extreme but fits.  The [5,9] upper path double SBJ is harder than that, so insane.  The springflings aren't bad but can make you drop the temp blue.

[5,7] starting from 2 is mostly reasonable.  Snails can get in the way, but you can avoid them.  Breaking the right speed blocks to clip up is a bit tricky, but fair for extreme.  An extreme trick into another extreme trick with jumpIntoCFClip is mean, but there is PB leniency for that ☘️

**Pull request merged**